data/json/obsoletion_and_migration_0.I/migration_items.json

```diff
@@ -90,6 +90,26 @@
     "replace": "box_medium_wood",
     "contents": [ { "id": "steel_lump", "count": 6 } ]
   },
+  {
+    "id": "tshirt_text",
+    "type": "MIGRATION",
+    "replace": "tshirt"
+  },
+  {
+    "id": "flag_shirt",
+    "type": "MIGRATION",
+    "replace": "tshirt"
+  },
+  {
+    "id": "linuxtshirt",
+    "type": "MIGRATION",
+    "replace": "tshirt"
+  },
+  {
+    "id": "tshirt_tour",
+    "type": "MIGRATION",
+    "replace": "tshirt"
```

Comment:
I think you can migrate them into specific corresponding variants here? At the least the flag, since it seems to be just one variant. I am pretty sure I did that with toys and the makeshift body pillow in the past.

Reply:
i can